zmon_worker_monitor/zmon_worker/tasks/main.py

```diff
@@ -1747,6 +1751,8 @@ def notify_for_trial_run(self, val, req, alerts, force_alert=False):
             try:
                 is_in_period = in_period(alert.get('period', ''))
             except InvalidFormat, e:
+                current_span.log_kv('exception', traceback.format_exc())
+                current_span.set_tag('error', True)
```

Comment:
Let's not set this as an error, I would rather add another tag, like, ``invalid_timeperiod=True`` so we can isolate those while searching.